### DISCUSSION TITLE
[Errno 2] No such file or directory: 'configuration.toml'

### DISCUSSION THREAD

**Discussion by Ondjultomte** [netbox-community/netbox#8588] [Q&A]
Hi, Im rtying netbox as a container and got it up and running with napalm too.

Im just cant find any documentation on the path were to place the confoguration.toml file.
I tried netbox "root" int /env in /configuration and after placing the toml file "everywhere" restarted netbox, but I still get the same error message.

**Comment by candlerb** [comment]
Netbox doesn't have a toml configuration file. It has "configuration.py", but that's already included in the docker container - if you're using the [community netbox-docker](https://github.com/netbox-community/netbox-docker) image - which in turn picks up settings from [environment variables](https://github.com/netbox-community/netbox-docker/blob/release/configuration/configuration.py).

Where are you seeing this error, and what exactly are you doing to cause the error to appear?

**Comment by Ondjultomte** [comment]
From the menu Other->Integrations->Scripts

![image](https://user-images.githubusercontent.com/16661547/153474279-13d510fb-88ea-4c83-b794-a85095afd62a.png)


![image](https://user-images.githubusercontent.com/16661547/153474426-6afac305-0056-408a-a559-639add2631af.png)

**Reply by candlerb** [reply]
This happens when you click the "Scripts" menu item on the LHS, correct?

I think it simply means one of the custom scripts you have installed is broken.  If you get a full backtrace it may tell you which one.

Otherwise, simply remove them from the `/opt/netbox/netbox/scripts/` directory one by one until you find the broken one. For more info see the [custom scripts](https://netbox.readthedocs.io/en/stable/customization/custom-scripts/) documentation.

To get a full backtrace, set EMAIL and ADMINS in configuration.py, and then the exception backtrace should be E-mailed to you when it happens.

**Reply by Ondjultomte** [reply]
ok, thnx!
